composer.json

```diff
@@ -25,7 +25,7 @@
     "issues": "https://github.com/TYPO3/testing-framework/issues"
   },
   "require": {
-    "phpunit/phpunit": "^7.5.6",
+    "phpunit/phpunit": "^8",
```

Comment:
we still could do php 7.5 + 8?

Reply:
aah, I meant "phpunit/phpunit:^7.5.7 || ^8" ?

Reply:
True, I adjusted the PR. No reason to hard require v8.